book/src/keymap.md

```diff
@@ -223,7 +223,7 @@ Jumps to various locations.
 | `.`   | Go to last modification in current file          | `goto_last_modification`   |
 | `j`   | Move down textual (instead of visual) line       | `move_line_down`           |
 | `k`   | Move up textual (instead of visual) line         | `move_line_up`             |
-
+| `w`   | Show labels at each word and select the word that belongs to the entered labels | `goto_word`
 #### Match mode
```

Comment:
```suggestion
| `w`   | Show labels at each word and select the word that belongs to the entered labels | `goto_word` |

#### Match mode
```

this probably isn't an issue while rendering the markdown but it looks like it's breaking github's markdown syntax highlighting a little